packages/sawja/sawja.1.5.12/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+maintainer: "sawja@inria.fr"
+authors: "Sawja development team"
+homepage: "http://sawja.inria.fr"
+bug-reports: "https://github.com/javalib-team/sawja/issues"
+license: "GPL-3.0"
+dev-repo: "git+https://github.com/javalib-team/sawja.git"
+build: [
+  ["./configure.sh"]
+  [make]
+]
+remove: [
+  ["ocamlfind" "remove" "sawja"]
+]
```

Comment:
This shouldn't be necessary as opam tracks files that have been installed during the installation
```suggestion
```